MediaBrowser.MediaEncoding/Probing/FFProbeHelpers.cs

```diff
@@ -16,24 +16,19 @@ public static void NormalizeFFProbeResult(InternalMediaInfoResult result)
                 throw new ArgumentNullException(nameof(result));
             }
 
-            if (result.format != null && result.format.tags != null)
+            if (result.Format != null && result.Format.Tags != null)
             {
-                result.format.tags = ConvertDictionaryToCaseInSensitive(result.format.tags);
+                result.Format.Tags = ConvertDictionaryToCaseInSensitive(result.Format.Tags);
```

Comment:
If you are renaming stuff, this should be `CaseInsensitive` and not `CaseInSensitive`